config/servicemappings/tags.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: core.cnrm.cloud.google.com/v1alpha1
+kind: ServiceMapping
+metadata:
+  name: tags.cnrm.cloud.google.com
+  namespace: cnrm-system
+spec:
+  name: Tags
+  version: v1beta1
+  serviceHostName: tags.googleapis.com
+  resources:
+    - name: google_tags_location_tag_binding
+      kind: TagsLocationTagBinding
+      idTemplate: "{{location}}/{{name}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      serverGeneratedIDField: name
+      resourceID:
+        targetField: name
+      resourceReferences:
+        - key: parentRef
+          tfField: parent
+          gvk:
+            kind: Project
+            version: v1beta1
+            group: resourcemanager.cnrm.cloud.google.com
+          targetField: number
+          valueTemplate: "//cloudresourcemanager.googleapis.com/projects/{{value}}"
+          parent: true
+        - key: tagValueRef
+          tfField: tag_value
+          gvk:
+            kind: TagsTagValue
```

Comment:
I do not think TagsTagValue is a resource KCC currently support, so we cannot reference to it. 
You may want to add TagsTagValue to service mapping as well, similar to EdgeContainerCluster: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/edgecontainer.yaml#L95

Reply:
[CRD](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/tags_v1beta1_tagstagvalue.yaml) is provisioned with KCC and [service mapping](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/resource-autogen/generated/servicemappings/tags.yaml) is also there. I tested [TagsTagBinding](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/tags_v1beta1_tagstagbinding.yaml) before working on LocationTagBinding for KCC and it was returning the expected error, so the assumption was TagValues and TagKeys are also supported.

Please confirm that they should be re-declared

Reply:
Thanks for the pointer! Yes they are auto-generated resources that's why I did not aware of it.